include/caf/leds.rst

```diff
@@ -69,54 +68,74 @@ To define PWM ports used by the application, complete the following steps:
 			ch2-pin = <7>;
 		};
 
-	Make sure to configure all PWM ports that are used by the application.
-	In particular, the number of PWM ports used must match the value set in :option:`CONFIG_CAF_LEDS_COUNT`.
+	Make sure to configure all PWM ports and channels that are used by the application.
 	For more help, see :ref:`zephyr:dt-guide`.
 
-2. Create a configuration file with the following array of arrays:
+2. Enable or create the ledpwm nodes in the devicetree file by setting the status to ``"okay"``. This can be done by using one of the following options:
 
-	* ``led_pins`` - contains PWM port and pin mapping for each LED used in the application.
+	* Enabling or create relevant nodes in a board-specific :file:`dts` file.
+	* Enabling or create relevant nodes in :file:`dts.overlay` file, which are to be merged to configuration.
 
-	The size of the array is determined by the :option:`CONFIG_CAF_LEDS_COUNT` and :option:`CONFIG_CAF_LEDS_COLOR_COUNT` Kconfig options.
 	The following code snippets show several examples of how the file contents could look:
 
 	* Example 1:
 
-	.. code-block:: c
+	.. code-block:: none
 
-		static const size_t led_pins[CONFIG_CAF_LEDS_COUNT]
-					    [CONFIG_CAF_LEDS_COLOR_COUNT] = {
-			{
-				DT_PROP(DT_NODELABEL(pwm0), ch0_pin)
-			}
+		&pwm_led0 {
+			status = "okay";
+			pwms = <&pwm0 15>;
+			label = "LED0";
 		};
 
 	* Example 2:
```

Comment:
Is this example valid for one of the options in lines 76-77? If so, which one?

Reply:
as above